### DISCUSSION TITLE
PyVista at OnScale

### DISCUSSION THREAD

**Discussion by akaszynski** [pyvista/pyvista#2466] [Show and tell]
It's probably common knowledge, but I work for [Ansys](https://www.ansys.com/), and we've recently acquired another company, [OnScale](https://www.ansys.com/news-center/press-releases/4-13-22-ansys-to-expand-cloud-portfolio-with-technology-from-acquisition-of-onscale). While looking at some of their tutorials and online documentation, I discovered that they (OnScale) use PyVista!

![image](https://user-images.githubusercontent.com/11981631/163398619-64b97aaf-76b5-43d4-8f34-48320a1b0252.png)

Just thought to let everyone know that PyVista being used everywhere! It seems to be a popular choice for computer aided engineering.

Here' the link to the [tutorial](https://onscale.com/blog/how-to-plot-data-arrays-using-the-onscale-solve-jupyter-notebook/
).

**Comment by tkoyama010** [comment]
I also work for a CAE (of structure) software vendor and am very interested in this topic. before PyVista was developed, the best visualization we could do in CAE was to force 3D in matplotlib or write long scripts in vtk. I think PyVista is the best solution for Python visualization in CAE.

**Reply by tkoyama010** [reply]
I love this service :)
![FRQsMiLaIAAN0_a](https://user-images.githubusercontent.com/7513610/165271598-2dc9d2ea-d08f-465a-8c88-6e7e53d4410e.jpeg)

**Reply by tkoyama010** [reply]
@akaszynski Can we use PyAnsys in this?

**Reply by akaszynski** [reply]
Soon... and pyvista + threejs (or vtk.js via panel) will be there as well at least in the post-processing section.
